### ISSUE TITLE
Thread - Thread title in report details RHP shows the entire message

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#53308]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.68-2
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Y
**Email or phone of affected tester (no customers):** applausetester+232803@applause.expensifail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to any chat.
3. Send a long message.
4. Right click on the message > Reply in thread.
5. Click on the thread header.

## Expected Result:
The thread title in report details RHP should have limit for the lines it will display.

## Actual Result:
The thread title in report details RHP shows the entire message and user has to scroll down to see the buttons.

## Workaround:
Unknown

## Platforms:

- [x] Android: Standalone
- [x] Android: HybridApp
- [x] Android: mWeb Chrome
- [x] iOS: Standalone
- [x] iOS: HybridApp
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/b696a2fe-f0d9-414b-ba7a-15b0ee4e7ccd
 
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
@twilight2294 are you in Slack?

**Comment by mountiny** [comment]
@twilight2294 its not clear what the fix is, we should discuss with the design team

**Comment by mountiny** [comment]
We will revert @twilight2294 and then you can handle this case in the PR again and discuss with the design team the best course of action, no need to rush to some solutions we do not have agreement on 👍

**Comment by mjasikowski** [comment]
Revert PR on the way

**Comment by mjasikowski** [comment]
Revert PR merged, CP request sent

**Comment by dannymcclain** [comment]
I agree with Shawn—2-3 lines seems totally reasonable to me. If we can't pick the amount of lines, I bet we can estimate a character length that gives us something similar.

**Comment by sonialiap** [comment]
I'm OOO Dec 16-20, adding a leave buddy

next steps:
@joekaufmanexpensify adding you directly since this cropped up from PR for https://github.com/Expensify/App/issues/52836. I don't believe a payment will be needed here because the original issue hasn't been paid but adding a buddy just in case

**Comment by joekaufmanexpensify** [comment]
Correct, this was a regression from my issue. No payment will be needed here.

**Comment by joekaufmanexpensify** [comment]
I actually think we can go ahead and close this one as it was a regression of https://github.com/Expensify/App/issues/52836 so no payment needed here. Will handle payment for the original issue there!
